incubator/kube-downscaler/templates/serviceaccount.yaml

```diff
@@ -1,11 +1,16 @@
-{{ if .Values.rbac.create -}}
+{{ if .Values.serviceAccount.create -}}
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: {{ include "kube-downscaler.fullname" . }}
+  name: {{ include "kube-downscaler.serviceAccountName" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
Is this line needed? 🤔 I would think it creates resources in the release namespace by default.

Reply:
That's an ongoing discussion for about a year or so: https://github.com/helm/helm/issues/5465
While in general helm is supposed to place items into the correct namespace, there are use-cases depending on namespace field to be present. I err on the side of caution here.

Reply:
🤷 fine; probably should be added to other resources for consistency, but I won't hold the PR based on this!